src/evaluate.jl

```diff
@@ -151,52 +109,57 @@ function evaluate!{T<:Number}(x::Array{Taylor1{TaylorN{T}},1}, δt::T,
     nothing
 end
 
-## Evaluates HomogeneousPolynomials and TaylorN on a val of the nv variable
-## using Horner's rule on the nv variable
-function horner{T<:Number,S<:NumberNotSeriesN}(a::HomogeneousPolynomial{T},
-        b::Tuple{Int,S} )
-    nv, val = b
-    numVars = get_numvars()
-    @assert 1 <= nv <= numVars
+"""
+    evaluate(a, vals)
+
+Evaluate a `HomogeneousPolynomial` polynomial at `vals`.
+"""
+function evaluate{T<:Number,S<:TaylorSeries.NumberNotSeriesN}(a::HomogeneousPolynomial{T},
```

Comment:
Here, you don't need `TaylorSeries.NumberNotSeriesN`; it's enough to have `NumberNotSeriesN`.